### DISCUSSION TITLE
Latest node version breaks Packer: C++ Assertion failed runtime error w node_modules\fibers

### DISCUSSION THREAD

**Discussion by qedric** [hayes0724/shopify-packer#134] [General]
I had to upgrade node for another project and when I tried to run packer start (after having to re-install packer) I got a C++ error, Running npm audit gave me a lot of high severity issues.

To get it working again, I first had to install a previous version of node:
nvm install --15.14.0

then switch to that version
nvm use 15.14.0

It would be great if packer were updated to work with the latest version so it's not necessary to switch versions and reinstall everything when switching between projects

Hopefully this save someone time if they run into the same issue ^^

**Comment by hayes0724** [comment]
This was required for fibers package when changing versions of npm it would require a reinstall of packer. Fibers has been removed in the latest version and it now uses dart-sass
